lightly/active_learning/config/sampler_config.py

```diff
@@ -21,6 +21,17 @@ class SamplerConfig:
             other attributes and the datetime. A new tag will be created in the
             web-app under this name.
 
+    Examples:
+        >>> # sample 100 images with CORESET sampling
+        >>> config = SamplerConfig(method=SamplingMethod.CORESET, n_samples=100)
+        >>> config = SamplerConfig(method='CORESET', n_samples=100)
```

Comment:
@MalteEbner do you know if this actually works?

Reply:
Yes, it looks well.